_pages/do-i-need-clearance/index.md

```diff
@@ -2,19 +2,43 @@
 title: Do I need clearance?
 permalink: /do-i-need-clearance/
 anchors:
-  - name: What type of information are you collecting?
   - name: Who are you collecting information from?
+  - name: What type of information are you collecting?
+  
 ---
 
 Not every request or collection falls under the PRA’s scope, and you may not need clearance at all. When figuring out if PRA clearance is needed, consider the following:
 
-- What type of information are you collecting
-- Who are you collecting it from
+- What type of information are you collecting,
+- Who are you collecting it from,
 - How will you be collecting it, and
-- Why are you collecting this information
+- Why you are collecting this information.
 
 This is just a starting point to see if your collection needs PRA clearance. We encourage [getting in touch with your agency's PRA contact]({{'/contact/' | relative_url}}) to answer more in-depth questions.
 
+### Who are you collecting information from?
+
+#### Members of the public
+
+Generally, this means people or groups outside of the federal government. Some groups that are considered members of the public include:
+
+- Individual people (including federal contractors)
+- Businesses and associations
+- State, tribal, and local governments
+
+If you’re only collecting information from federal employees or military personnel as part of their job, then you don’t need PRA clearance. If the information isn’t part of their work-related duties, you will need PRA clearance.
+
+#### Ten or more people or groups
+
+Over a 12-month period, if you are requesting the same information from ten or more people or entities, you need PRA clearance.
+
+If you’re requesting information from fewer than ten people or groups, but they represent the majority or all of an industry or sector, you will need PRA clearance.
+
+#### Voluntary or Mandatory Collection
```

Comment:
A few thoughts on this, since I know a few people were tripped up by this "voluntary language".

Should this be under "what type of information are you collecting"?

Should we be more direct and just give this a heading "voluntary collections are not exempt"

Reply:
I dig this!